src/test/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketIntegrationClientFactory.java

```diff
@@ -127,7 +128,7 @@ private BitbucketClouldIntegrationClient(String payloadRootPath, String owner, S
         }
 
         @Override
-        protected CloseableHttpResponse executeMethod(HttpRequestBase httpMethod) throws InterruptedException, IOException {
+        protected CloseableHttpResponse executeMethod(HttpHost host, HttpRequestBase httpMethod) throws InterruptedException, IOException {
```

Comment:
host is not used?

Reply:
it is used in the real client, but not in the mock class. But if the signatures do not match, tests bypass the mock and hit the actual client (which is both unintended and fails)